.github/workflows/iroha2-release.yml

```diff
@@ -52,22 +52,39 @@ jobs:
           cache-from: type=gha
           cache-to: type=gha,mode=max
 
-  load-rs:
+  configs:
     runs-on: ubuntu-latest
     container:
       image: hyperledger/iroha2-ci:nightly-2022-12-22
+    steps:
+      - uses: actions/checkout@v3
+        ref: iroha2-dev
+      - name: Setup git config
+        run: |
+          git config user.name "GitHub Actions Bot"
+          git config user.email "<>"
```

Comment:
is this ok for DCO rules? IMO it should be. We could replace with a dummy address like [here](https://github.com/orgs/community/discussions/41101)